src/main/java/edu/hm/hafner/analysis/IssueBuilder.java

```diff
@@ -28,14 +30,26 @@
  */
 @SuppressWarnings({"InstanceVariableMayNotBeInitialized", "JavaDocMethod", "PMD.TooManyFields"})
 public class IssueBuilder {
+    private static final String EMPTY = StringUtils.EMPTY;
+    private static final String UNDEFINED = "-";
+    private static final TreeString UNDEFINED_TREE_STRING = TreeString.valueOf(UNDEFINED);
+    private static final TreeString EMPTY_TREE_STRING = TreeString.valueOf(StringUtils.EMPTY);
+
+    private TreeStringBuilder fileNameBuilder = new TreeStringBuilder();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'fileNameBuilder' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=4885606)